Python/ast_unparse.c

```diff
@@ -76,7 +76,7 @@ append_ast_boolop(_PyUnicodeWriter *writer, expr_ty e, bool omit_parens)
 static int
 append_ast_binop(_PyUnicodeWriter *writer, expr_ty e, bool omit_parens)
 {
-    const char *op;
+    const char *op = NULL;
```

Comment:
You cannot pass ``NULL`` to ``append_charp`` because ``_PyUnicodeWriter_WriteASCIIString`` doesn't handle NULL as second argument.